src/integTest/groovy/nebula/plugin/plugin/NebulaPluginPluginPublishingIntegrationSpec.groovy

```diff
@@ -0,0 +1,51 @@
+package nebula.plugin.plugin
+
+class NebulaPluginPluginPublishingIntegrationSpec extends GitVersioningIntegrationSpec {
+
+    @Override
+    def setupBuild() {
+        buildFile << """
+        apply plugin: 'nebula.plugin-plugin'
+        """
+    }
+
+    def 'publish snapshot'() {
+        when:
+        def result = runTasksSuccessfully('snapshot', '-PnetflixOss.username=user',  '-PnetflixOss.password=password', '--dry-run')
+
+        then:
+        result.standardOutput.contains(':publishPluginMavenPublicationToNetflixOSSRepository SKIPPED')
+    }
+
+    def 'candidate task invokes publication to NetflixOSS repository'() {
+        when:
+        git.add(patterns: ['.'] as Set)
+        git.commit(message: 'Setup 2')
```

Comment:
These tests that publish snapshot, then candidate, then release in order are very cool 👍